packages/in_app_purchase/in_app_purchase/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -247,12 +255,10 @@
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-Runner/Pods-Runner-resources.sh",
 				"${PODS_CONFIGURATION_BUILD_DIR}/in_app_purchase_storekit/in_app_purchase_storekit_privacy.bundle",
-				"${PODS_CONFIGURATION_BUILD_DIR}/shared_preferences_foundation/shared_preferences_foundation_privacy.bundle",
```

Comment:
This is expected: `in_app_purchase_storekit` hasn't been migrated to SPM yet, but `shared_preferences_foundation` has.